### PR TITLE
New comment styling

### PR DESCRIPTION
**Pull request by kaedroho**

### CODE REVIEW

File: src/main.scss [hunk 1]
```diff
@@ -48,8 +48,8 @@ $color-state-live-draft: #43b1b0;
 $color-box-background: $color-white;
 $color-box-border: $color-grey-3;
 $color-box-text: $color-black;
-$color-textarea-background: $color-white;
-$color-textarea-border: $color-grey-3;
+$color-textarea-background: #f2fcfc;
```
Comment (emilytoppm) [inline src/main.scss]: Aren't we trying to reduce the number of shades of colours in the Wagtail Admin?

File: src/components/CommentReply/index.tsx [hunk 2]
```diff
@@ -199,7 +197,7 @@ export default class CommentReplyComponent extends React.Component<CommentReplyP
 
     return (
       <>
-        {this.renderAuthorDate()}
+        {this.renderHeader()}
```
Comment (emilytoppm) [inline src/components/CommentReply/index.tsx]: It does feel like there's a lot of duplication in these methods for the `Comment` and `CommentReply` components - feels like we could get away with just changing the classname and contents of the final div on most

**Review by emilytoppm — COMMENTED** [review]
Looking good so far! Still need to try it out with Wagtail once there's a corresponding PR, but the code is looking nice - just a minor concern about moving loads of new shades of grey into the admin, but if that's the design...

My other comment is just general really - I think `CommentHeader` is definitely helping here, but it's nice to reduce some of the duplication in the components.

File: src/main.scss (hunk 1 above)
Comment (kaedroho) [reply src/main.scss]: I think we can pick this up in design review

**Review by emilytoppm — APPROVED** [review]
LGTM!

**Pull request merged**